src/app/components/driver-list/driver-list.component.ts

```diff
@@ -116,11 +114,11 @@ getGoogleApi()  {
  */
 displayRoute(origin, destination, service, display) {
     service.route({
-      origin: origin,
-      destination: destination,
+      origin,
+      destination,
       travelMode: 'DRIVING',
       //avoidTolls: true
```

Comment:
Commented out line

Reply:
should remove commented out lines from legacy code?
this one in particular came with the code

Reply:
I would say yes, remove any legacy commented code.  We aren't going to use it, and it's just a code smell.